app/src/main/java/org/wikipedia/analytics/eventplatform/ImageRecommendationsEvent.kt

```diff
@@ -0,0 +1,41 @@
+package org.wikipedia.analytics.eventplatform
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import org.wikipedia.WikipediaApp
+
+@Serializable
+@SerialName("/analytics/mobile_apps/android_image_recommendation_event/1.0.0")
+class ImageRecommendationsEvent(
+   private val action: String,
+   private val active_interface: String,
+   private val action_data: String,
+   private val primary_language: String,
+   private val wiki_id: String
+) : MobileAppsEvent(STREAM_NAME) {
+
+    companion object {
+        private const val ACTION_IMPRESSION = "impression"
+        private const val STREAM_NAME = "eventlogging_EditAttemptStep"
+
+        fun logImpression(activeInterface: String) {
+            submitImageRecommendationEvent(ACTION_IMPRESSION, activeInterface, "", "")
+        }
+
+        fun logAction(action: String, activeInterface: String, actionData: String, wikiId: String) {
+            submitImageRecommendationEvent(action, activeInterface, actionData, wikiId)
+        }
+
+        fun getActionDataString(filename: String = "", recommendationSource: String = "", recommendationSourceProject: String = "",
```

Comment:
If the `recommendationSourceProject` is supposed to be similar to the [Growth schema](https://gerrit.wikimedia.org/r/plugins/gitiles/schemas/event/secondary/+/refs/heads/master/jsonschema/analytics/mediawiki/structured_task/article/image_suggestion_interaction/current.yaml#92), then:

- It should be plural `recommendationSourceProjects`
- The value should come from the `GrowthImageSuggestion.ImageItem.projects` list, and _not_ the current language wiki.
- The "projects" list basically says "We're recommending this image because it was also used in x, y, z language wikipedia."